rom/src/rom.rs

```diff
@@ -516,3 +555,28 @@ pub fn rom_start(params: RomParameters) {
         }
     }
 }
+
+#[derive(Debug, Default)]
+pub struct AxiUsers {
+    pub mbox_users: [Option<u32>; 5],
+    pub fuse_user: u32,
+    pub trng_user: u32,
+    pub dma_user: u32,
+}
+
+impl<'a> From<&'a StaticRef<McuStraps>> for AxiUsers {
```

Comment:
It feels a little weird to be taking `&'a` of a `StaticRef`. Why not just `impl From<&'static McuStraps> for AxiUsers`?